Dockerfile-dev

```diff
@@ -12,11 +9,13 @@ RUN apk add --no-cache gcc musl-dev linux-headers git
 
 COPY requirements.txt .
 COPY requirements_production.txt .
+COPY cleanup.sh .
+COPY setup.cfg .
 
 RUN pip install --no-cache-dir --requirement requirements.txt
 
-COPY openslides_backend/ ./openslides_backend/
-COPY setup.cfg .
+ARG OPENSLIDES_BACKEND_ACTION_PORT=9002
```

Comment:
Why dont you have to copy the code into the container?

Reply:
Because you mount the directories in the makefile with the `-v` switch. `docker run -v foo:/srv/code/bar` automatically creates the folder `/srv/code/bar` in your container and mounts the folder `foo` there.